trello_utils.py

```diff
@@ -0,0 +1,52 @@
+import os
+import requests
+
+TRELLO_KEY = os.environ.get('TRELLO_KEY')
+TRELLO_TKN = os.environ.get('TRELLO_TKN')
+
+TRELLO_BOARD = '5f48fa1737fa77134ffe1e17'
+TRELLO_TODO_LIST = '5f48fa17b70abe80f7f0942d'
+TRELLO_DONE_LIST = '5f48fa17d5f15e744c86a725'
+TRELLO_URL_BASE = 'https://api.trello.com/1/'
+
+CARD_DONE_STATUS = 'Done!'
+CARD_TODO_STATUS = 'To-do'
+
+LOCAL_CARDS = []
+
+
+def trimCardsList(trello_cards):
+    for card in trello_cards:
+        updateOrAddCard(Card(
+            card['id'],
+            card['name'],
+            getCardStatus(card['id'])
+        ))
+    print LOCAL_CARDS.sort(key=cardComparator)
```

Comment:
Looks like you're using Python 2
`print` is a function in Python 3 and this line throws an error because it should be e.g. `print(LOCAL_CARDS.sort(key=cardComparator)`

You should be running flask from inside the virtual env to ensure everyone is using the same versions of things when they run this app. All you need to do is run the setup command in the README whenever you open a new terminal, before you run `flask run`. (Behind the scenes it is running either `source env/scripts/activate` or `source env/bin/activate` depending on whether you're on Windows)